packages/angular/projects/clr-angular/src/timeline/_timeline.clarity.scss

```diff
@@ -17,8 +17,7 @@
     min-width: $clr-timeline-step-min-width;
     margin-left: $clr-timeline-gutter-width;
 
-    cds-icon,
-    clr-icon {
+    cds-icon:not([size]) {
```

Comment:
I don't think this is the right approach. With this solution, users have to set the size manually on each icon used in the timeline content. The culprit of the problem is, I believe, is that the style defined for the timeline description icon size is leaking into the timeline content. That's why the icons in the content also getting as big as timeline description icons. So the correct approach I think is to improve the target specificity of that timeline description icon size and prevent it from affecting other areas.

Reply:
The selector could be `> cds-icon, > clr-icon` and we need both!

Reply:
I don't think we need both cds-icon and clr-icon any more. The template was upgraded to use cds-icon. Since our template isn't using clr-icon anymore we shouldn't need to target it. If consumers are using clr-icon then we don't risk affecting it. 

@shijir i think this fix is ok b/c it establishes a 'default' icon size, the same as what we use in the template. If the consumer wants to use cds/clr-icon they can set the size thats right for their content and our style won't affect it.

Reply:
@hippee-lee We do need both. https://github.com/vmware/clarity/pull/5579

Reply:
As I understand it, this sizing style should only apply to icons in the template and we only use cds-icon elements in the templates. Am I missing something?